sycl/doc/extensions/experimental/sycl_ext_oneapi_annotated_ptr.asciidoc

```diff
@@ -603,14 +603,16 @@ return tmp;
 a|
 [source,c++]
 ----
-T operator++();
-T operator++(int);
-T operator--();
-T operator--(int);
+T operator++() const;
+T operator++(int) const;
+T operator--() const;
+T operator--(int) const;
 ----
 |
-increment and decrement operator of annotated_ref. The annotations are applied
-when the object `T` is loaded and stored to the memory.
+Increment and decrement operator of annotated_ref. Increment/Decrement the object `T`
+referenced by this wrapper via `T`'s Increment/Decrement operator.
```

Comment:
```suggestion
referenced by this wrapper via ``T``'s Increment/Decrement operator.
```

The HTML formatting is broken here.  I think it will be fixed by using the double-backtick here instead of single-backtick.  You need to use double-backtick when code-font is in the middle of a word